tsdb/importer/import-trial.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
this looks like a duplicate file.

Reply:
Yes, this most definitely is, I will remove it!